apps/projects/app/script.js

```diff
@@ -128,13 +128,25 @@ async function handleEvents(response) {
   case 'RepoUpdated':
     console.log('[Projects] RepoUpdated', response.returnValues)
     nextState = await syncRepos(appState, response.returnValues)
+  case 'AssignmentApproved':
```

Comment:
I was able to crash the app with the following error message in the console after approving an assignment during the test script: `getRepoData failed:  TypeError: Cannot read property 'request' of undefined` 

Not sure why the client variable declared above would fail to be initialized